Sources/MongoSwift/MongoConnectionString.swift

```diff
@@ -154,6 +168,27 @@ public struct MongoConnectionString: Codable, LosslessStringConvertible {
                     self.authMechanism = try MongoCredential.Mechanism(value)
                 case "authmechanismproperties":
                     self.authMechanismProperties = try self.parseAuthMechanismProperties(properties: value)
+                case "ssl":
+                    self.ssl = try value.getBool(forKey: "ssl")
+                case "tls":
+                    self.tls = try value.getBool(forKey: "tls")
+                case "tlsallowinvalidcertificates":
```

Comment:
since these constants are used in a few places, I think it would be beneficial to define them as Swift constants to help prevent accidental typos.

Reply:
+1